### PR TITLE
Include files necessary to run MSVC build

### PR DESCRIPTION
**Pull request by StephenCWills**

### CODE REVIEW

**Review by AJenbo — APPROVED** [review]
fun

**Comment by StephenCWills** [comment]
What in the world happened? Half the build have failed downloads.

File: .github/workflows/Windows_MSVC_x64.yml [hunk 1]
```diff
@@ -33,22 +33,31 @@ jobs:
...
+        path: |
+            build-ninja-vcpkg-relwithdebinfo/devilutionx.exe
+            build-ninja-vcpkg-relwithdebinfo/bz2.dll
+            build-ninja-vcpkg-relwithdebinfo/fmt.dll
+            build-ninja-vcpkg-relwithdebinfo/libpng16.dll
+            build-ninja-vcpkg-relwithdebinfo/libsodium.dll
+            build-ninja-vcpkg-relwithdebinfo/SDL2.dll
+            build-ninja-vcpkg-relwithdebinfo/SDL2_image.dll
+            build-ninja-vcpkg-relwithdebinfo/zlib1.dll
+            build-ninja-vcpkg-relwithdebinfo/devilutionx.mpq
```
Comment (StephenCWills) [inline .github/workflows/Windows_MSVC_x64.yml:63]: The build I ran on my fork earlier suggests that devilutionx.mpq is not available on the MSVC build.
https://github.com/StephenCWills/devilutionX/actions/runs/4046760328

**Comment by StephenCWills** [comment]
The artifact seems to have everything you need to run the game except devilutionx.mpq. Compared to the MinGW build, it's missing the license files, and it's not double-zipped. I assumed the latter would be a good thing, but it seems to trip a security feature in Chrome that detects the exe inside the zip file and tried to block the download. 🤷

File: .github/workflows/Windows_MSVC_x64.yml [hunk 2]
```diff
@@ -33,22 +33,30 @@ jobs:
...
-        path: build-ninja-vcpkg-debug/devilutionx.exe
+        path: |
+            build-ninja-vcpkg-relwithdebinfo/devilutionx.exe
+            build-ninja-vcpkg-relwithdebinfo/bz2.dll
+            build-ninja-vcpkg-relwithdebinfo/fmt.dll
+            build-ninja-vcpkg-relwithdebinfo/libpng16.dll
+            build-ninja-vcpkg-relwithdebinfo/libsodium.dll
+            build-ninja-vcpkg-relwithdebinfo/SDL2.dll
+            build-ninja-vcpkg-relwithdebinfo/SDL2_image.dll
+            build-ninja-vcpkg-relwithdebinfo/zlib1.dll
```
Comment (glebm) [inline .github/workflows/Windows_MSVC_x64.yml:62]: This supports globs, so you can simply do:
```
build-ninja-vcpkg-relwithdebinfo/*.dll
```

**Pull request merged**